prebid-mobile/code-integration-android.md

```diff
@@ -71,31 +71,25 @@ adUnits.add(adUnit2);
 
 Once configuration is done, use the following API to initialize Prebid Mobile and start fetching Prebid ads for your list of ad units.
 
-The following two APIs are being deprecated:
+Please use the one below for initialization:
```

Comment:
We should make this match below to point out that this one is for DFP. Change to "If you're using DFP as your primary ad server, use the API like this:"